packages/core/src/components/account/MyAccountMenu/MyAccountMenu.tsx

```diff
@@ -0,0 +1,85 @@
+import { Button, Avatar, Link } from '@faststore/ui'
+import { useRouter } from 'next/router'
+
+import styles from '../section.module.scss'
+
+import useScreenResize from 'src/sdk/ui/useScreenResize'
+
+interface NavItem {
+  title: string
+  route: string
+  isSelected?: boolean
+}
+
+export interface MyAccountMenuProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  avatarImageUrl?: string
+  accountName: string
+  items: NavItem[]
+}
+
+const AvatarContainer = ({
+  imageUrl,
+  name,
+}: { imageUrl?: string; name: string }) => (
+  <div className={styles.avatarContainer}>
+    <Avatar imageUrl={imageUrl} alt={name}></Avatar>
```

Comment:
If the avatar has no children. You can do it like this:
```suggestion
    <Avatar imageUrl={imageUrl} alt={name} />
```